UPGRADE-3.0.md

```diff
@@ -1,6 +1,11 @@
 UPGRADE FROM 2.x to 3.0
 =======================
 
+Compatibility
+-------------
+
+Using `doctrine/cache` is no longer possible. Migrate to PSR-6 instead.
```

Comment:
I think this isn't specific enough. What isn't possible? Surely users can continue using doctrine/cache, they just cannot configure doctrine caching options to use services binding to it directly.